virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar/launchbar.component.ts

```diff
@@ -75,6 +125,55 @@ export class LaunchbarComponent implements MVDHosting.LogoutActionInterface {
      });
    }
 
+  ngOnInit() {
+    this.themeService.onColorChange
+      .subscribe((color:any) => {
+        this._theme.color.windowColorActive = this.adjustColorByLightness(color.themeColor, 20);
+        this._theme.color.windowTextActive = color.textColor;
+        this._theme.color.launchbarColor = this.adjustColorByLightness(color.themeColor, -20)+'b2'; // Adds some transparency to bottom app bar
+        this._theme.color.launchbarText = color.textColor;
+        this._theme.color.launchbarMenuColor = color.themeColor;
+        this._theme.color.launchbarMenuText = color.textColor;
+
+        this.changeTheme.emit(this._theme);
+    });
+
+    this.themeService.onSizeChange
+      .subscribe((size:any) => {
+        this._theme.size.window = size.windowSize;
+        this._theme.size.launchbar = size.launchbarSize;
+        this._theme.size.launchbarMenu = size.launchbarMenuSize;
+
+        this.changeTheme.emit(this._theme);
+    });
+
+    this.themeService.onResetAllDefault
+      .subscribe(() => {
+        this.resetThemeDefault();
+      });
+  }
+
+  resetThemeDefault(): void { // Current default theme is dark grey, with light text
+    this._theme.size.window = DEFAULT_SIZE;
+    this._theme.size.launchbar = DEFAULT_SIZE;
+    this._theme.size.launchbarMenu = DEFAULT_SIZE;
+
+    this._theme.color.windowColorActive = Colors.COOLGREY_80;
+    this._theme.color.windowTextActive = "#f4f4f4";
+    this._theme.color.launchbarColor = "#0d0d0e"+'b2';
```

Comment:
It looks like an invalid color.

Reply:
If you're referring to the "b2" portion, that is an addition to make the 0d0d0e shade of grey more transparent, however I'm pretty sure f4f4f4 is a color that's part of the "Colors" palette so I should add that in anyway

Reply:
Looks like it wasn't, cool

Reply:
ok, `b2` is alpha.